src/main.ts

```diff
@@ -11,21 +11,31 @@ export async function run(): Promise<void> {
     const labelName = core.getInput('label_name', {required: true})
     const labelColor = core.getInput('label_color', {required: true})
     const loginsToIgnore = core.getInput('logins_to_ignore', {required: false})
+    const failIfMember = core.getInput('fail_if_member', {required: false})
     const orgs = getOrgMembershipList()
     const token = core.getInput('token', {required: true})
-
     const client = new GitHubClient(token)
-
-    if (client.hasLabel(labelName)) {
-      core.info(`The '${labelName}' label is already present on this issue! 🙌`)
-      return
-    }
+    const labels = client.getLabels()
+    const hasCommunityLabel = labels && !labels.includes(labelName)
 
     if (
       (await client.checkOrgMembership(orgs)) ||
       client.isExcludedLogin(loginsToIgnore)
     ) {
-      core.info("Looks like this issue doesn't need labeling! 👍")
+      if (failIfMember === 'true' && !labels) {
+        core.setFailed(
+          'The PR does not have the required label, please add it!'
+        )
+      } else {
+        core.info("Looks like this issue doesn't need labeling! 👍")
+      }
+      return
+    }
+
+    if (hasCommunityLabel) {
+      core.setFailed(
+        'Community PR must have a label name community, please add it!'
```

Comment:
```suggestion
        'Community PRs must be labelled!'
```

Reply:
so here community pr's are labelled automatically this error is if the label is anything other than community

Reply:
yeah doesnt really make much sense.
`Community PR must have a label name community`

Reply:
done

Reply:
done